os_migrate/playbooks/delete_conversion_hosts.yml

```diff
@@ -1,3 +1,15 @@
+- hosts: os_migrate_conv_src
```

Comment:
The src/dst hosts won't be in the inventory at this point. We'll need to first include the conversion host role with `tasks_from: conv_hosts_inventory.yml` from the migration host, that's what creates the conversion host inventory entries.

(I guess this wasn't tested? It would be nice to test this at least manually before merging.)

Reply:
Not tested yet.  Wanted to get a review (glad I did).  My assumption is that this would only be testable via e2e, so that was my plan on where to test.